stream_chat/async_chat/client.py

```diff
@@ -45,7 +45,8 @@ def __init__(
             api_key=api_key, api_secret=api_secret, timeout=timeout, **options
         )
         self.session = aiohttp.ClientSession(
-            connector=aiohttp.TCPConnector(keepalive_timeout=3.9)
+            base_url=self.base_url,
```

Comment:
how about regular client?

Reply:
can’t be done there as far as I digged into urllib3

Reply:
himm I think we can specify timeout on session to long version (59s) and on retrieved conn per call to `this.timeout` (6s) with retry in place, should be good for it

Reply:
This is all what we can specify on HTTP adapter level: ([link](https://github.com/psf/requests/blob/main/requests/adapters.py#L114))

```
    :param pool_connections: The number of urllib3 connection pools to cache.
    :param pool_maxsize: The maximum number of connections to save in the pool.
    :param max_retries: The maximum number of retries each connection
        should attempt. Note, this applies only to failed DNS lookups, socket
        connections and connection timeouts, never to requests where data has
        made it to the server. By default, Requests does not retry failed
        connections. If you need granular control over the conditions under
        which we retry a request, import urllib3's ``Retry`` class and pass
        that instead.
    :param pool_block: Whether the connection pool should block for connections.
```

Retry logic we already have one:
```py
        self.session = requests.Session()
        self.session.mount("http://", requests.adapters.HTTPAdapter(max_retries=1))
        self.session.mount("https://", requests.adapters.HTTPAdapter(max_retries=1))
```

Reply:
It's a parameter for session `requests.Session(timeout=timedelta(seconds=59))` 

I believe this would work.